configuration/pih/styles/global/zl.css

```diff
@@ -1,83 +1,83 @@
 header {
-	background: #FF9800;
+    background: #FF9800;
```

Comment:
And the same question here... this looks to be shifting the css spacing from 6 spaces to 4 spaces (I think?).  Let's standardize... but defer to @louidorjp  and @brandones  on what they want to use.